tests/_dockerfiles/ubuntu-16.04/Dockerfile

```
@@ -0,0 +1,25 @@
+FROM ubuntu:16.04
+ARG AWK
+ARG NODE_VERSION=16
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get -y update && \
+    apt-get -y install \
```

How about the following code?

```suggestion
ENV DEBIAN_FRONTEND noninteractive
RUN apt-get -y update && \
    apt-get -y install \
```
or
```suggestion
RUN apt-get -y update && \
    DEBIAN_FRONTEND=noninteractive apt-get -y install \
```

---

Doesn't `export DEBIAN_FRONTEND=noninteractive` affect the whole container image?

---

No. Every `RUN` instruction creates an intermediate container and its shell environment is ephemeral (not committed).